Automattic/pocket-casts-android
# Updated empty state in now playing screen

## Description

Updates the `NowPlayingPager.kt` to handle empty state of the player. Also avoids the 💥crash that happens when user previously clicked on the area where the episode name would be.

Fixes #1094

## Testing Instructions
1. Open the app with an account that has no podcasts currently playing or in the up next queue.
2. Swipe left to get to the Now Playing screen
3. ✅ Verify that the player shows the message "Pick an episode to begin" in the title.
4. ✅ Verify that the play/rewind/forward buttons are disabled and greyed out.
5. ✅ Verify that clicking anywhere on the player doesn't crash the app.

## Screenshots or Screencast 
<img src="https://github.com/Automattic/pocket-casts-android/assets/89896473/594f7e6e-de3d-4d5b-b366-918f2ffbf1cb" width="40%" height="40%">


## Checklist
 
#### I have tested any UI changes...
